.pre-commit-config.yaml

```diff
@@ -11,11 +11,6 @@ ci:
     autoupdate_schedule: monthly
 
 repos:
--   repo: https://github.com/python-poetry/poetry
```

Comment:
is there a hook to validate pyproject.toml for hatch ? Or at least that it follows pyproject specifications?

In the past, we got a broken pyproject, not detected in the repo itself since the package itself is not installed in the github workflow, but it broke another repo that was depending on it.

It makes me think that it could be useful if the workflows were installing the package. Which might happen anyway when we move to pixi...
Ok good for now, we can revisit this aspect later

Reply:
Not really. Hatch doesn't have a `check` command, like poetry to verify its pyproject.toml.

Maybe, we could create a command test pyproject we just run one basic test, to check if the project could be build.

Reply:
We could also run `hatch build` to build the project, but it seems a lot of spent time before each commit.

Reply:
If the doesn't have it, we need to delete the following `test_version_is_semver`. Indeed, it used `geoh5py.__version__` variable. However, this variable is not defined.

Reply:
version should still be semver. And Analyst expects `__version__`

I think __init__.py needs:
```py
from geoh5py._version import __version__

__all__ = [
    __version__
]
```

Maybe a try catch around the import with that `RuntimeErrror("geoh5py package is not correctly installed. Please install through pip or Conda.")`

Reply:
I updated it the `pyproject.toml` with `versioningit` instead of `hatch-vcs`. It offers us the possibility to format the `__version__` variable. Without it, we don't pass `test_version_is_semver` (at least locally).